Description: Debian's adduser doesn't allow uppercase letters by default
Author: Robert Ancell <robert.ancell@canonical.com>
Forwarded: no
Comment: Save the associated string break for Ubuntu 17.04

Index: gnome-control-center-3.20.1/panels/user-accounts/um-utils.c
===================================================================
--- gnome-control-center-3.20.1.orig/panels/user-accounts/um-utils.c
+++ gnome-control-center-3.20.1/panels/user-accounts/um-utils.c
@@ -548,7 +548,6 @@ is_valid_username (const gchar *username
                  */
                 for (c = username; *c; c++) {
                         if (! ((*c >= 'a' && *c <= 'z') ||
-                               (*c >= 'A' && *c <= 'Z') ||
                                (*c >= '0' && *c <= '9') ||
                                (*c == '_') || (*c == '.') ||
                                (*c == '-' && c != username)))
